ledger/src/use_snapshot_archives_at_startup.rs

```diff
@@ -31,7 +35,11 @@ pub mod cli {
         and will only use snapshot-related state already on disk. \
         If there is no state already on disk, startup will fail. \
         Note, this will use the latest state available, \
-        which may be newer than the latest snapshot archive.";
+        which may be newer than the latest snapshot archive. \
+        Specifying \"when-newest\" will use snapshot-related state \
+        already on disk unless there are snapshot archives newer. \
```

Comment:
a bit awkward wording here: "unless there are snapshot archives newer."
"newer than it"?